### DISCUSSION TITLE
Make QInfiniteScroll  work in both directions simultanously?

### DISCUSSION THREAD

**Discussion by lyle45** [quasarframework/quasar#9311] [General - Components / Directives / etc]
I opened this feature recently but hadn't been able to figure it out

Is there a way currently to make infinite scroll work in both directions control the reverse prop dynamically ( isn't enough on it's own)? Will most likely start in the middle of the scroll area

https://github.com/quasarframework/quasar/issues/9159

**Comment by lyle45** [comment]
bump?

**Reply by hamidhosseini** [reply]
I need this as well. Did you find anything?

**Reply by lyle45** [reply]
@hamidhosseini already implemented this for my need
You can take the component from my pr. Works 100%

https://github.com/quasarframework/quasar/pull/9848

You can also mark thumbs up for this and the pr so they take it to master
